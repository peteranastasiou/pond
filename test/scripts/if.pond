
# TODO with functions all cases can be tested

var a = 1000;
if( a==5 ){
    print("A" + a);
} else if( a==15 ) {
    print("B" + a);
} else {
    print("C" + a);
}

if( a > 15 ){
    print("a is big");
}

if( a < 15 ){
    print("a is small");
}

