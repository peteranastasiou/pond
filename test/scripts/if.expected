C1000
a is big
